app.js

```diff
@@ -1,2 +1,114 @@
-/* add your code here */
+/*add code here*/
+document.addEventListener('DOMContentLoaded', () => {
+    const stocksData = JSON.parse(stockContent);
+    const userData = JSON.parse(userContent);
 
+    generateUserList(userData, stocksData);
+
+    const saveButton = document.getElementById('btnSave');
+    const deleteButton = document.getElementById('btnDelete');
```

Comment:
For selectors, we need to preface id or class 
```suggestion
    const saveButton = document.getElementById('#btnSave');
    const deleteButton = document.getElementById('#btnDelete');
```